builds/fabric8-che/plugins/keycloak-plugin-server/src/main/java/com/redhat/che/keycloak/server/KeycloakUserChecker.java

```diff
@@ -39,7 +40,7 @@
     private static final String ENDPOINT = "http://che-host:8080/api/token/user";
 
     @Inject
-    private KeycloakHttpJsonRequestFactory requestFactory;
+    private DefaultHttpJsonRequestFactory requestFactory;
```

Comment:
so I think it should be clearly stated in the comment ? Because I guess people will try to remove implem and use interface.